.claude/hooks/session-context.sh

```
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Hook E: Inject useful context at session start
+# Event: SessionStart (startup)
+
+PROJECT_DIR="${CLAUDE_PROJECT_DIR:-$(git rev-parse --show-toplevel 2>/dev/null)}"
+[ -z "$PROJECT_DIR" ] && exit 0
+cd "$PROJECT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)